src/test/java/uk/nhs/prm/deduction/e2e/models/ContinueRequestMessage.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.prm.deduction.e2e.models;
+
+import uk.nhs.prm.deduction.e2e.utility.Resources;
+
+import java.util.UUID;
+
+import static uk.nhs.prm.deduction.e2e.utility.TestUtils.getUUIDAsUpperCaseString;
+
+public class ContinueRequestMessage {
+    private UUID conversationId;
+    private UUID messageId;
+    private String sourceGpOds;
+    private String destinationGpOds;
+    private String sourceGpAsid;
+    private String destinationGpAsid;
+
+    public ContinueRequestMessage(UUID conversationId, UUID messageId, String sourceGpOds, String destinationGpOds, String sourceGpAsid, String destinationGpAsid) {
+        this.conversationId = conversationId;
+        this.messageId = messageId;
+        this.sourceGpOds = sourceGpOds;
+        this.destinationGpOds = destinationGpOds;
+        this.sourceGpAsid = sourceGpAsid;
+        this.destinationGpAsid = destinationGpAsid;
+    }
+
+    public String toJsonString() {
```

Comment:
Could we rename this to `getJsonString`? :D

```suggestion
    public String getJsonString() {
```

Reply:
For this one I think I am keeping it as `toJsonString` for now, because there are two other classes written by TW which use the name `toJsonString` for the same logic.
Later after we moved packages to main directory, then we could rename all those methods in one go.

Reply:
No problem :)